pkg/stdlib/op.cue

```diff
@@ -65,7 +65,7 @@ import (
 #ApplyComponentRemaining: #Steps & {
```

Comment:
This is recommended to be deprecated in the future. /cc @wangyikewxgm

Reply:
Ok, this wfStep isn't used in any addon already.